components/automate-grpc/protoc-gen-policy/policy/policy.go

```diff
@@ -48,56 +46,28 @@ type policyBundle struct {
 	validate  policyInfoValidator
 }
 
-// policyVersions lets us differentiate between "v1" and "v2" policies, which
-// each use their own specific "policy package" and annotations.
-var policyVersions = map[string]policyBundle{
-	"v1": {
-		pkg:       "github.com/chef/automate/components/automate-gateway/api/authz/policy",
-		extension: policy.E_Policy,
-		convert: func(x interface{}) *policyInfo {
-			if y, ok := x.(*policy.PolicyInfo); ok {
-				return &policyInfo{action: y.Action, resource: y.Resource}
-			}
-			return nil
-		},
-		validate: func(msg pgs.Message, pi *policyInfo) error {
-			expanded, err := dummyExpand(msg, pi.resource)
-			if err != nil {
-				return err
-			}
-			// borrowing IsAuthorizedReq's protobuf validation to ensure
-			// our resource and action annotations are correct.
-			req := &authz.IsAuthorizedReq{
-				Subjects: []string{"user:local:albertine"}, // this won't fail validation
-				Resource: expanded,                         // resource and
-				Action:   pi.action,                        // action could
-			}
-			return req.Validate()
-		},
+var policyVersion = policyBundle{
+	pkg:       "github.com/chef/automate/components/automate-gateway/api/iam/v2/policy",
```

Comment:
Formerly `policy_v2`, this has now become just `policy` as the leaf node of the path. In use, this translates to code now using `policy.thing` instead of `policyv2.thing` as you will see in various files.